targets/lenovo-x1/appvms/element.nix

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
```

Comment:
This file should be probably renamed to dendrite.nix or dendire-pinecone.nix?

Reply:
No, it is going to be element.nix because element and dendrite should run together in the same VM.